Autopilot/IMU/Src/IMUInterface.cpp

```diff
@@ -0,0 +1,8 @@
+#include "IMUInterface.h"
```

Comment:
I don't like the idea of creating a folder specifically for the IMU. Just toss these files in inc and src. Maybe some day we can dedicate a folder to drivers and there interfaces.

Reply:
Agreed. Driver files are starting to really pile up

Reply:
In fact, let's add this to this doc: https://uwarg-docs.atlassian.net/l/c/va1g0kvi